Create CONTRIBUTING.md (#81)
thamara/time-to-leave: CONTRIBUTING.md

# Contributing to Time to Leave

> Hey there! We are so happy you are interested in contributing to TTL. 

Reviewer: ```suggestion
Hey there! We are so happy you are interested in contributing to TTL.

```

Thank you in advance for your help! :)

> ### When you open an issue or a bug report

Reviewer: ```suggestion
### When you open an issue
```
Seemed quite redundant previously, don't you think?

> When opening an issue, please provide a good description of the issue you are facing, or the feature you'd like to have.

Reviewer: ```suggestion
When opening an issue, please provide a good description of the problem you are facing, or the feature you'd like to have.

```

> For bugs, it's good to provided some step-by-step to reproduce te issue, as well as some screenshots, if appliable. It's also important to inform the operation system you are using and the version of TTL.

Reviewer: ```suggestion
For bugs, it's good to provide some step-by-step instructions to reproduce the issue, as well as some screenshots, if applicable. It's also important to inform the operating system you are using and the version of TTL.
```

### When you open a pull request

> All PRs are very welcome, and we'll do our best to review it as fast as possible. 

Reviewer: ```suggestion
All PRs are very welcome, and we'll do our best to review them as fast as possible.

```

> If you want to work on a specific issue, just shout out to us and we'll assign the issue to you. Once you are done, just submit a PR with a relevant description of the change, and provide screenshots if appliable.

Reviewer: ```suggestion
If you want to work on a specific issue, just shout out to us and we'll assign the issue to you. Once you are done, just submit a PR with a relevant description of the change, and provide screenshots if applicable.
```

> One tip: Before pushing the PR you can run ```npm run lint``` or ```npm run lint-fix``` to avoid having failures on our continuous integration.

Reviewer: ```suggestion
*Tip*: Before pushing the PR you can run ```npm run lint``` or ```npm run lint-fix``` to avoid having failures on our continuous integration.
```
